bugbug/bug_features.py

```diff
@@ -463,44 +463,46 @@ def __call__(self, bug, **kwargs):
 
 class couple_common_whiteboard_keywords(couple_bug_feature):
     def __call__(self, bugs, **kwargs):
-        return [
-            keyword
-            for keyword in whiteboard_keywords(bugs[0])
-            if keyword in whiteboard_keywords(bugs[1])
-        ]
+        return len(
+            [
+                keyword
+                for keyword in whiteboard_keywords(bugs[0])
+                if keyword in whiteboard_keywords(bugs[1])
+            ]
+        )
 
 
 class is_same_product(couple_bug_feature):
     def __call__(self, bugs, **kwargs):
-        return bugs[0]["product"] == bugs[1]["product"]
+        return int(bugs[0]["product"] == bugs[1]["product"])
```

Comment:
Why an int here and in the other features?

Reply:
OK, read your earlier comment and I replied in https://github.com/mozilla/bugbug/pull/889#issuecomment-670501994 :)